docs/developers/Custom_Rules.mdx

```diff
@@ -305,7 +305,6 @@ This rule bans for-of looping over an enum by using the TypeScript type checker
 
 ```ts
 import { ESLintUtils } from '@typescript-eslint/utils';
-import * as tsutils from 'ts-api-utils';
```

Comment:
There's really no need to ask users to load in a full third-party-package for a single enum helper function. Especially not for a package outside of the org's ownership.